### ISSUE TITLE
Disable security by default in websocket-source

### ISSUE THREAD

**Issue by sabbyanandan** [spring-cloud-stream-app-starters/websocket#5]

**Comment by mminella** [comment]
Can we set this to be profile driven?  AKA off by default unless the cloud profile is on for example (so it's on by default in CF).

**Comment by artembilan** [comment]
Sure! Anything is possible with Spring!

The question is only what is the difference between running the app locally or in CF?
It is still exposes a particular socket and ready to accept requests.

The idea to disable security has come exactly from the integration testing side for SCDF, therefore I think CF is included into the suit as well.

I might miss something, but hope @sabbyanandan  will help us to see the reasonable ceiling for this requirement.

Thanks

**Comment by mminella** [comment]
Locally, I'd expect to be behind some form of firewall/etc.  On CF, you very well may be exposed to the internet.  I'm just an overly paranoid person who prefers secure by default...

**Comment by artembilan** [comment]
Right, everybody does the same (especially Spring Boot 2.0), but requirements from the customer - first!
